### PR TITLE
Publish RDS/Aurora logs to Cloudwatch Logs

### PR DESCRIPTION
**Pull request by ringods**
Add support for the property `enabled_cloudwatch_logs_exports` which allows for automatic publishing of MySQL/PostgreSQL/Aurora logs to Cloudwatch Logs. Pointers to the respective AWS documentation pages in README.md

My changes are backwards compatible and only need a minor version bump.

### CODE REVIEW

**Comment by ringods** [comment]
Ping @simonrondelez / @duboisph

File: README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ Creates a RDS instance, security_group, subnet_group and parameter_group
 | availability\_zone | The availability zone where you want to launch your instance in | string | `""` | no |
 | backup\_retention\_period | How long do you want to keep RDS backups | string | `"14"` | no |
 | default\_parameter\_group\_family | Parameter group family for the default parameter group, according to the chosen engine and engine version. Defaults to mysql5.7 | string | `"mysql5.7"` | no |
+| enabled\_cloudwatch\_logs\_exports | List of log types to enable for exporting to CloudWatch logs. You can check the available log types per engine in the [AWS RDS documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_LogAccess.html#USER_LogAccess.Procedural.UploadtoCloudWatch). Defaults to nothing. | list | `[]` | no |
```
Comment (duboisph) [inline README.md:19]: Can you make sure this documentation is also added in full to the output's description? We use [terraform-docs](https://github.com/segmentio/terraform-docs) to generate the inputs/outputs documentation.

**Review by duboisph — APPROVED** [review]
LGTM, just a small documentation comment.

File: README.md (hunk 1 above)
Comment (iuriaranda) [reply README.md:19]: you probably mean the "variable's description" 😛

**Review by iuriaranda — APPROVED** [review]
Besides @duboisph comment, LGTM

File: README.md (hunk 1 above)
Comment (duboisph) [reply README.md:19]: lol yes 😅

Comment (ringods) [reply README.md:19]: @duboisph @iuriaranda updated the variable description.

**Comment by duboisph** [comment]
@ringods thanks for the contribution. [It's available as release `4.3.0`](https://github.com/skyscrapers/terraform-rds/releases/tag/4.3.0)

**Pull request merged**